components/Chat/ChatMessage.tsx

```diff
@@ -212,6 +209,28 @@ export const ChatMessage: FC<Props> = memo(
                         </td>
                       );
                     },
+                    li({ children }) {
+                      const colorMatch =
```

Comment:
This is how I can modify the code to check if it contains a code element and then check for the hex code. Is it ok to add here then we will get the color highlight if AI returns the hex color in code element or ``

``` typescript
const regex = /#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})/g;
  let colorMatch = regex
    .exec(
      String(children),
    );

  if (children?.length > 1) {
    Children.forEach(children, (child) => {
      if (isValidElement(child) && child.props.node.tagName === 'code') {
        console.log({ child });
        const match = regex.exec(child.props.children);
        if (match) {
          colorMatch = match;
        }
      }
    })
  }
```